modules/mapper-extras/build.gradle

```diff
@@ -29,3 +31,8 @@ restResources {
     includeCore '_common', 'cluster', 'nodes', 'indices', 'index', 'search', 'get'
   }
 }
+if (BuildParams.inFipsJvm){
+  // The cluster fails to form with a trial license and we can't test FIPS 140 mode with a basic license
```

Comment:
The description was vague because I hadn't figured why either. I reached out to core-features and this is now resolved in https://github.com/elastic/elasticsearch/pull/65971, so I will amend this